include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -2,15 +2,70 @@
 
 #include<iostream>
 #include<pjsua2.hpp>
+#include<unordered_map>
+#include <string>
 
 #include"Softphone.hpp"
+#include"SoftphoneArguments.hpp"
 
 class SoftphoneManager
 {
 public:
-    SoftphoneManager();
-    ~SoftphoneManager();
+    SoftphoneManager(int number_of_softphones, int port, std::string domain):
+        _port(port), _domain(std::move(domain))
+    {
+        pjLibraryInit();
+    }
+
+    ~SoftphoneManager() = default;
```

Comment:
Add the call to `libDestroy` in the destructor

Reply:
done